public/app/plugins/panel/geomap/layers/data/markersLayer.tsx

```diff
@@ -88,16 +93,27 @@ export const markersLayer: MapLayerRegistryItem<MarkersConfig> = {
     if (config.showLegend) {
       legend = <ObservablePropsWrapper watch={legendProps} initialSubProps={{}} child={MarkersLegend} />;
     }
-    const shape = markerMakers.getIfExists(config.shape) ?? circleMarker;
+
+    let shape = markerMakers.getIfExists(config.shape) ?? circleMarker;
```

Comment:
nit: we should try to avoid `let` if possible. Makes the code hard to follow and complex.